src/lib/blockchain/insuranceOracle.ts

```
@@ -10,11 +10,11 @@ const supabase = createClient(
  * Fetches real-time weather data from NOAA API (mocked for this implementation).
  * In production, this would be a Chainlink External Adapter or dedicated Oracle node.
  */
-export async function fetchNOAAWeatherData(lat: number, lon: number): Promise<number> {
+export async function fetchNOAAWeatherProbability(lat: number, lon: number): Promise<number> {
     // Mock NOAA API response
```

ESLint

[error] Delete `··`

(prettier/prettier)